plugins/accounts/plugins/accounts/less/hawkular-accounts.less

```diff
@@ -11,3 +11,7 @@
   color: #fff;
   font-weight: bold;
 }
+
+body {
+  padding-top: 0;
+}
```

Comment:
For some reason, the toastr.css causes the body to have a padding-top of 20px (IIRC). So, this is to reset that...